apps/server/src/modules/fwu-learning-contents/fwu-learning-contents.config.ts

```diff
@@ -0,0 +1,17 @@
+import { Configuration } from '@hpi-schul-cloud/commons';
+import { S3Config } from './interface/config';
+
+export const s3Config: S3Config = {
+	endpoint: Configuration.get('FWU_CONTENT__S3_ENDPOINT') as string,
+	region: Configuration.get('FWU_CONTENT__S3_REGION') as string,
+	bucket: Configuration.get('FWU_CONTENT__S3_BUCKET') as string,
+	accessKeyId: Configuration.get('FWU_CONTENT__S3_ACCESS_KEY') as string,
+	secretAccessKey: Configuration.get('FWU_CONTENT__S3_SECRET_KEY') as string,
+};
+
+const fwuLearningContentsConfig = {
+	NEST_LOG_LEVEL: Configuration.get('NEST_LOG_LEVEL') as string,
+	INCOMING_REQUEST_TIMEOUT: Configuration.get('FWU_CONTENT__INCOMING_REQUEST_TIMEOUT') as number,
```

Comment:
as i said it in some of your other prs, if you are dealing with time, add a unit to your variable